src/analysis/special_functions/log/base.lean

```diff
@@ -52,6 +52,16 @@ by simp_rw [logb, log_div hx hy, sub_div]
 
 @[simp] lemma logb_inv (x : ℝ) : logb b (x⁻¹) = -logb b x := by simp [logb, neg_div]
 
+@[simp] lemma inv_logb {a b : ℝ} : (logb a b)⁻¹ = logb b a := by simp_rw [logb, inv_div]
```

Comment:
This one strikes me as a slightly questionable simp lemma; If I'm proving something about `logb 2 x`, I think making a `logb x 2` appear in my goal state would be confusing and surprising!

So maybe to be safe,
```
lemma inv_logb {a b : ℝ} : (logb a b)⁻¹ = logb b a := by simp_rw [logb, inv_div]
```

Reply:
Yeah, I agree, that could be confusing